src/components/resume-builder/SectionSelectionCards.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+// HOOKS
+import { useLayout } from './context/LayoutContext';
+
+// COMPONENTS
+import { TbCircleCheckFilled } from 'react-icons/tb';
+
+// TYPES
+import type { ActiveSectionName } from './types/layout';
+
+// CONFIGS
+import { SectionIdTitleMapping } from './config/section-name-config';
+
+const sectionsNameList = Object.keys(SectionIdTitleMapping).filter(
+  (sectionName) => sectionName.length,
+) as ActiveSectionName[];
+
+export const SectionSelectionCards = () => {
+  const { sectionsOrder, updateSectionsOrder } = useLayout();
+
+  const onSelectionCardClick = (
```

Comment:
Here 1 comment can be given to explain why this method has arguments and how it is planned to execute.